app/models/payment.rb

```diff
@@ -0,0 +1,40 @@
+class Payment < ApplicationRecord
+  # See: https://github.com/rails/rails/issues/31234#issuecomment-446629054
+  # Because of the open enum-state a payment needs to be able to be in (Mollie requirement),
+  # we're undefining #open before defining it with the enum.
+  class << self
+    undef_method :open
+  end
+  # See: https://docs.mollie.com/payments/status-changes
+  enum status: { open: 0, pending: 1, paid: 2, failed: 3, canceled: 4, expired: 5 }
+  COMPLETE_STATUSES = %w[paid failed canceled expired].freeze
+
+  belongs_to :user
+
+  validates :user, :status, presence: true
+  validates :amount, numericality: { greater_than_or_equal_to: 20 }, presence: true
```

Comment:
You can still create a mollie payment of less than 20 euros right now. You are redirected to Mollie with the entered amount of less that 20 euros but the Payment object is then not saved and then you get a Routing Error when you go back to Sofia.